android/src/main/java/com/rnfs/RNFSManager.java

```diff
@@ -324,6 +355,20 @@ public void copyFileAssets(String assetPath, String destination, Promise promise
   }
 
   @ReactMethod
+  public void copyFileRes(String filename, String destination, Promise promise) {
+    String prefix = filename.substring(filename.lastIndexOf(".")+1);
+    String name = filename.substring(0, filename.lastIndexOf("."));
+    Boolean isImage = prefix.equals("png") || prefix.equals("jpg") || prefix.equals("jpeg") || prefix.equals("bmp") || prefix.equals("gif") || prefix.equals("webp") || prefix.equals("psd") || prefix.equals("svg") || prefix.equals("tiff");
```

Comment:
This code exists 3 times in the file now. In order to reduce duplicated code it would be better to have an own static method here called "isImage" that would be called from all 3 occurrences where this detection is needed